docs/components/webapp.rst

```diff
@@ -54,12 +54,9 @@ superuser.
 
 Run this::
 
-  $ docker-compose run webapp python manage.py createsuperuser
+  $ docker-compose run webapp python manage.py makesuperuser email@example.com
```

Comment:
Cool! I was able to use that and it actually worked. 
Also, rebased this up. Will land this PR once travis is happy.